src/entity_recognizer/EntityFeatures.cpp

```diff
@@ -24,6 +24,7 @@
 void EntityFeatures::AddUnigramFeatures(SequenceInstanceNumeric *sentence,
```

Comment:
Don't add space between ) and ? for statements like x = (condition)? 0 : 1

Lines with more than 80 columns are supposed to be split.